from dpctl.tensor._dlpack import DLPackCreationError
from numpy.exceptions import AxisError

__all__ = [
    "AxisError",
    "DLPackCreationError",
]
